### ISSUE TITLE
Screen readers fail to recognize UI elements when I build the engine myself

### ISSUE THREAD

**Issue by EtiTheSpirit** [godotengine/godot#112671]
### Tested versions

v4.6.dev.custom_build [e47fb8b89] (the test build I cobbled together for this report)
v4.5.1.stable.mono.conservatory_edition [6d6165f55] (my own fork, see issue description)

### System information

Godot v4.6.dev (e47fb8b89) - Windows 10 (build 19045) - Multi-window, 3 monitors - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 3070 Ti (NVIDIA; 32.0.15.8108) - Intel(R) Core(TM) i9-10900K CPU @ 3.70GHz (20 threads) - 31.85 GiB memory

### Issue description

**For some vital context ahead of time:** 
* I first noted this issue in my own build of the engine, **which is running on 4.5.1-stable. The issue was first noted when my fork was based on 4.5, however.** 
* At that time, I chose to not report the issue, because I only noticed it in my fork. Accessibility tools functioned as intended when I used versions of the engine I downloaded from the website.
* **Vitally,** I did not ever touch accessibility code at all in my fork, so this bug was a bit strange to me!

Zoom forward to today, where I need to focus heavily on accessibility as part of my UI cleanup process. Just kind of by chance, I got a bit of a hunch: What would happen if I built the engine plain and stock, right here, right now?

Lo and behold, after building the repo (as of [e47fb8b89]), narrators and other screen readers have no context into what's going on inside of the game window, the same exact issue I noticed in my own fork. To complete the test, I downloaded 4.6-dev3 from the website, and just like last time, it *does* function in the officially provided build.

To really triple check, I had a look with Accessibility Insights for Windows which can debug accessibility nodes in any application.

**`v4.6.dev.custom_build` and `v4.5.1.stable.mono.conservatory_edition`:**

Erroneous; Nothing shows up and the game is just considered to be a pane.

<img width="2647" height="1232" alt="Image" src="https://github.com/user-attachments/assets/b8a8b539-97ef-483c-b163-9440bfd90e39" />

***

**`v4.6-dev3` (from website under experimental builds):**

Correct; Buttons in the game window are registered as accessibility objects.

<img width="2656" height="1263" alt="Image" src="https://github.com/user-attachments/assets/a48b74ab-82d7-49dd-9539-228351f61413" />

***

I know this issue isn't exactly the most useful as it's basically a reverse *"It works on my machine"*, but I figured it was reproducible enough to the point where I ought to just try to report it anyway.

### Steps to reproduce

The steps I did are as follows: 
1. Open Visual Studio 2022
2. Click the "Clone a Repository..." button on the right side of the landing pad.
3. Enter this repository's git URL `https://github.com/godotengine/godot.git`
4. Clone the repo, let it finish downloading and such
5. Exit Visual Studio 2022
6. Open Powershell or Command Prompt in the repo directory
7. Execute `scons platform=windows` and wait for completion
8. Open `bin` directory and run the engine
9. Create a blank project, and a new scene for UI.
10. Add a button to the blank UI, give it some text.
11. Open the Windows Narrator or other accessibility application of your choosing.
12. Save the UI scene and set it as the startup scene. Run the project.
13. The narrator and other apps will be unable to recognize elements within the game window as accessibility objects, and won't read them aloud.

### Minimal reproduction project (MRP)

[accessibility-bug-report.zip](https://github.com/user-attachments/files/23499739/accessibility-bug-report.zip) 

\* This contains the scene outlined in the reproduction, just with two buttons so that I could focus them with tab and the arrow keys.

**Comment by bruvzg** [comment]
You need to download AccessKit libraries separately if you are using custom build.

- Download latest release from https://github.com/AccessKit/accesskit-c/releases or https://github.com/godotengine/godot-accesskit-c-static/releases (includes some additional builds for MinGW/LLVM and ARM, otherwise the same), or build it yourself.
- Extract the archive.
- Build Godot with `accesskit_sdk_path=PATH_TO_EXTRACTED_ARCHIVE` added to build command.

**Comment by bruvzg** [comment]
Unrelated note: This is also true for some other parts of the engine:
- D3D12 support - https://github.com/godotengine/godot-nir-static/, build option to set path `mesa_libs=PATH`
- ANGLE (OpenGL over D3D11 for old intel GPUs) - https://github.com/godotengine/godot-angle-static/, build option `angle_libs=PATH`.

**Comment by bruvzg** [comment]
You can also place `accesskit.dll` (from the same archive) alongside executable, this should work without any extra build flags. But might cause issues due to different C-runtime used by engine and library, and is in general less tested way to do it.

**Comment by EtiTheSpirit** [comment]
Excellent. Providing the SDK did indeed solve the issue, and I've taken the liberty of integrating the other modules into my build too, figured I might as well cover as many bases as I can since I want to be as broadly compatible as I can manage, but I digress. Thank you for your help.

Since this resolved my issue (after a looooooong rebuild) I'll go ahead and close this as resolved.
